src/Workspaces/Core/Portable/LanguageServices/SyntaxFactsService/ISyntaxFactsService.cs

```diff
@@ -250,6 +250,8 @@ internal interface ISyntaxFactsService : ILanguageService
 
         bool IsAnonymousFunction(SyntaxNode n);
 
+        bool IsAnonymousOrLocalFunction(SyntaxNode n);
```

Comment:
this is not generally what we put in ISyntaxFacts.  We put the low level primitives, which can be implemented by both langauges.  As there is already IsAnonymousFunction here, you should only add IsLocalFunction.

Then, your feature can make it's own IsAnonymousOrLocalFunction(ISyntaxFacts) function.

Reply:
➡️ Fixed in commit 38db528